src/styles/index.scss

```diff
@@ -1,3 +1,52 @@
+@import '/src/styles/variables';
+
 body {
+  @include center-all;
+
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((7 * $field-side) + (6 * $calendar-gap));
+  gap: $calendar-gap;
+
+  @each $start-day, $count in $days {
+    &--start-day-#{$start-day} :first-child {
+      margin-left: calc(($field-side + $calendar-gap) * $count);
+    }
```

Comment:
In these lines, you're applying a left margin to the first child of the calendar element based on the day of the week. The margin calculation seems to be correct, but the selector might not work as expected. The ':first-child' pseudo-class should be placed right after the parent selector without any space. Please change '&--start-day-#{$start-day} :first-child' to '&--start-day-#{$start-day}:first-child'.